ci-operator/config/openshift-pipelines/release-tests/openshift-pipelines-release-tests-release-v1.11__openshift-pipelines-ocp4.14-lp-rosa-hypershift.yaml

```diff
@@ -0,0 +1,58 @@
+base_images:
+  upi-installer:
+    name: "4.14"
+    namespace: ocp
+    tag: upi-installer
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.18
+images:
+- context_dir: .
+  dockerfile_path: Dockerfile
+  to: openshift-pipelines-runner
+releases:
+  latest:
+    candidate:
+      product: ocp
+      stream: ci
+      version: "4.14"
+resources:
+  '*':
+    limits:
+      memory: 8Gi
+    requests:
+      cpu: "1"
+      memory: 4Gi
+tests:
+- as: openshift-pipelines-rosa-hypershift
+  cron: 0 6 25 12 *
+  steps:
+    cluster_profile: aws-cspi-qe
+    env:
+      FIREWATCH_CONFIG: |
+        [
+          {"step": "aws-*", "failure_type": "all", "classification": "Infrastructure AWS", "jira_project": "LPTOCPCI", "jira_additional_labels": ["4.14-lp","rosa-hypershift-lp","pipelines-lp"]},
+          {"step": "gather-*", "failure_type": "all", "classification": "Other", "jira_project": "LPTOCPCI", "jira_additional_labels": ["4.14-lp","rosa-hypershift-lp","pipelines-lp"]},
+          {"step": "osd-*", "failure_type": "all", "classification": "User Setup", "jira_project": "LPTOCPCI", "jira_additional_labels": ["4.14-lp","rosa-hypershift-lp","pipelines-lp"]},
+          {"step": "rosa-*", "failure_type": "all", "classification":  "Cluster Creation", "jira_project": "LPTOCPCI", "jira_additional_labels": ["4.14-lp","rosa-hypershift-lp","pipelines-lp"]},
+          {"step": "cucushift-hypershift-*", "failure_type": "all", "classification": "Cluster Health Check", "jira_project": "LPTOCPCI", "jira_additional_labels": ["4.14-lp","rosa-hypershift-lp","pipelines-lp"]},
+          {"step": "openshift-pipelines-install-and-tests", "failure_type": "pod_failure", "classification": "Test Failure", "jira_project": "LPINTEROP", "jira_additional_labels": ["4.14-lp","rosa-hypershift-lp","pipelines-lp","interop-tests"]},
+          {"step": "openshift-pipelines-install-and-tests", "failure_type": "test_failure", "classification": "Test Failure", "jira_project": "SRVKP", "jira_component": "QA", "jira_additional_labels": ["4.14-lp","rosa-hypershift-lp","pipelines-lp","interop-tests"]}
+        ]
+      FIREWATCH_DEFAULT_JIRA_PROJECT: LPTOCPCI
+      FIREWATCH_JIRA_SERVER: https://issues.redhat.com
+      OCM_LOGIN_ENV: staging
+      OPENSHIFT_VERSION: "4.14"
+      REGION: us-west-2
```

Comment:
I was wondering if REGION is needed?

Reply:
I think @ascerra can comment better on this, but I think it's to keep all hypershift clusters in one region just for segregation purposes